public/js/main.js

```diff
@@ -0,0 +1,107 @@
+// Banner Carousel
```

Comment:
same file exists in different folder. If this is the compiled output file then output should go to a folder like dist/build etc. and do not commit those compiled files